docs/set_up_svs.rst

```diff
@@ -42,14 +42,6 @@ click **Add**, then click **Start Tails**.
 
 We will now prepare the *Secure Viewing Station*.
 
-Ensure Filenames are Preserved
-------------------------------
-
-In order to preserve filenames when you decrypt submissions, on each *Secure
-Viewing Station*, you should open a **Terminal** and type the following commands:
-
-.. include:: includes/tails-svs-nautilus.txt
-
 Correct the System Time
```

Comment:
Screenshot in this section is Tails3-specific. Could be updated.

Reply:
Thanks for taking care of that one, LGTM